### PR TITLE
doc: add option `include_req_body`

### LINKED ISSUE #2195: doc(kafka-logger): option `include_req_body`
The plugin "kafka-logger" already supports this feature, but we did not write this feature in the documentation.

https://github.com/apache/apisix/blob/master/apisix/plugins/kafka-logger.lua#L47

welcome PR ^_^

### PR DESCRIPTION
**Pull request by Yiyiyimu**
### What this PR does / why we need it:
fix #2195 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by Yiyiyimu** [comment]
Chinese doc of plugins format needs to be unified。

File: doc/plugins/syslog.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ This will provide the ability to send Log data requests as JSON objects.
 |max_retry_times|optional       |Max number of retry times after a connect to a log server failed or send log messages to a log server failed.|
 |retry_interval|optional       |The time delay (in ms) before retry to connect to a log server or retry to send log messages to a log server, default to 100 (0.1s).|
 |pool_size    |optional       |Keepalive pool size used by sock:keepalive. Default to 10.|
+|include_req_body|optional      |Return request body besides head; default is false|
```
Comment (membphis) [inline doc/plugins/syslog.md:50]: "Return request body besides head;"
->
"Whether to include the request body, the default is false, which means it is not included."

**Review by membphis — CHANGES_REQUESTED** [review]

File: doc/zh-cn/plugins/http-logger.md [hunk 2]
```diff
@@ -46,6 +46,7 @@
 | buffer_duration |可选的|必须先处理批次中最旧条目的最长期限（以秒为单位），默认值为 5|
 | max_retry_count |可选的|从处理管道中移除之前的最大重试次数，默认为 0|
 | retry_delay |可选的|如果执行失败，则应延迟执行流程的秒数，默认为 1|
+|include_req_body|可选的| 返回请求 body，默认为 false|
```
Comment (membphis) [inline doc/zh-cn/plugins/http-logger.md:49]: take a look at: https://github.com/apache/apisix/pull/2205#discussion_r486103749

**Review by membphis — COMMENTED** [review]
missing test case

**Pull request closed without merging**